### ISSUE TITLE
Sections of the website

### ISSUE THREAD

**Issue by seandavi** [NIHDataScienceSIG/nihdatasciencesig.github.io#2]
Besides talks/events, are there sections and information that you thing we should include on the website? We have the option of individual pages, groups of pages, or even pulling data from spreadsheets or text files to autogenerate content. Just jot down thoughts. What gets done will depend on availability of data and/or time. I just want to make a first pass at organizing.

**Comment by informationista** [comment]
a past events page that links to the OSF/Youtube content

**Comment by seandavi** [comment]
- Sponsored vents
   - Talks
   - Hackathons
   - Classes
- Partner events

**Comment by informationista** [comment]
maybe a resources page that people could contribute to, with links to interesting stuff?

**Comment by seandavi** [comment]
What about a "people" section (driven by pull requests)? Or is that too weird?

**Comment by informationista** [comment]
I think if we did a people section that was automatically populated, we'd need to make sure it was clearly explained how to get on and off of it

**Comment by seandavi** [comment]
Agreed. Perhaps we can table the idea for now, then, and just focus on "resources".
